src/schemas/json/workflows.json

```
@@ -259,6 +259,48 @@
         },
         "for": {
           "$ref": "#/definitions/for"
+        },
+        "parallel": {
+          "type": "object",
+          "description": "Run branches or iterations in parallel",
+          "additionalProperties": false,
+          "properties": {
+            "exception_policy": {
+              "description": "determines action of other branches when an exception occurs",
+              "type": "string",
+              "enum": ["continueAll"]
+            },
+            "shared": {
+              "description": "list of shared variables",
```

```suggestion
              "description": "A list of shared variables",
```

Optional fix: sentence start consistency violated: `A list` may sound better.